### PR TITLE
Installing via YAML

### PR DESCRIPTION
**Pull request by yhontyk**

### CODE REVIEW

File: modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-pipelines-operator	
+  namespace: openshift-marketplace
+spec:
+  targetNamespaces:
+  - openshift-marketplace
+----
```
Comment (nikhil-thomas) [inline modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc]: we don't need this step. As our operator is installed in AllNamespace install mode, the operatorgroup will be all ready present in `openshift-operators` namespace.

When installing from ui the namespace is automatically set to openshift-operators (cannot change for AllNamespace install mode)

Comment (yhontyk) [reply modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc]: Removed!

File: modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  sourceNamespace: openshift-marketplace
+----
+
+. Create the Subscription object:
++
+----
+$ oc apply -f sub.yaml
+----
++
+Openshift Pipelines are now installed in the target namespace.
```
Comment (nikhil-thomas) [inline modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc]: nit:
How about something like
"Openshift Pipelines are now installed in the default target namespace (openshift-pipelines)."

File: modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+.Prerequisites
+
+- Access to an OpenShift cluster using an admin account.
+
+- *oc* is installed on your system.
+
+.Procedure
+
+. Create a Subscription object YAML file to subscribe a namespace to the OpenShift Pipelines Operator,
+for example `sub.yaml`:
```
Comment (Preeticp) [inline modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc]: Comma after example

File: modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc [hunk 4]
```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * assembly_installing_pipelines.adoc
+
+[id="proc_installing-pipelines-operator-via-yaml.adoc_{context}"]
+= Installing OpenShift Pipelines from the OperatorHub using the CLI
```
Comment (Preeticp) [inline modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc]: Suggestion rename this to: Installing the OpenShift Pipelines operator using the CLI

File: modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+You can install OpenShift Pipelines from the OperatorHub using the CLI.
```
Comment (Preeticp) [inline modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc]: OpenShift Pipelines operator

File: modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+  sourceNamespace: openshift-marketplace
+----
+
+. Create the Subscription object:
++
+----
+$ oc apply -f sub.yaml
+----
++
+Openshift Pipelines are now installed in the default target namespace `openshift-pipelines`.
```
Comment (Preeticp) [inline modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc]: The OpenShift Pipelines operator is now...

File: modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc (hunk 2 above)
Comment (yhontyk) [reply modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc]: Fixed. Thank you.!

File: modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc (hunk 3 above)
Comment (yhontyk) [reply modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc]: Fixed in https://github.com/openshift/pipelines-docs/pull/57/commits/dd55a60f9f465192dfb3f4fd371d87d76df1c7db

File: modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc (hunk 4 above)
Comment (yhontyk) [reply modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc]: Fixed in https://github.com/openshift/pipelines-docs/pull/57/commits/dd55a60f9f465192dfb3f4fd371d87d76df1c7db

File: modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc (hunk 5 above)
Comment (yhontyk) [reply modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc]: Fixed in https://github.com/openshift/pipelines-docs/pull/57/commits/dd55a60f9f465192dfb3f4fd371d87d76df1c7db

File: modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc (hunk 6 above)
Comment (yhontyk) [reply modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc]: Fixed in https://github.com/openshift/pipelines-docs/pull/57/commits/dd55a60f9f465192dfb3f4fd371d87d76df1c7db

**Comment by yhontyk** [comment]
@Preeticp Renamed the files. And I needed to rebase because of all of those commits by Joan. Feel free to squash all of the commits on merge.

**Pull request merged**